src/InstrumentationEngine/InstructionGraph.cpp

```diff
@@ -281,8 +292,6 @@ HRESULT MicrosoftInstrumentationEngine::CInstructionGraph::DecodeInstructions(_I
     // Second pass, go backwards through the instructions and fix up the next pointers and finalize branch instructions
```

Comment:
This comment needs to be updated since you moved the SetNextInstruction from the first pass to the second pass.